RELEASE.md

```diff
@@ -62,7 +62,12 @@ Bump version
     $ sed -r -i "s/TOPOLVM_VERSION := [[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+/TOPOLVM_VERSION := ${VERSION}/g" example/Makefile
     $ sed -r -i "s/checkout v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+/checkout v${VERSION}/g" example/README.md
     ```
-4. Edit `CHANGELOG.md` for the new version ([example][]).
+4. Edit `CHANGELOG.md` for the new version ([example][]).  
+   The list of relevant PRs can be obtained by the following command.
+   ```
+   git log --merges --format="%s %b" $(git tag | grep "^v" | sort -V -r | head -n 1)..main | awk '/^Merge pull request #/ {c="";for(i=7;i<=NF;i++) c=c $i " "; printf "- %s([%s](https://github.com/topolvm/topolvm/pull/%s))\n", c, $4, substr($4, 2, 20);}' | tac
```

Comment:
```suggestion
   git log --merges --format="%s%x00%b" $(git tag | grep "^v" | sort -V -r | head -n 1)..main |sed -E 's|^Merge pull request #([0-9]*)[^\x0]*(.*)|- \2 ([#\1](https://github.com/topolvm/topolvm/pull/\1))|' | tac
```

As awk recognizes multiple spaces as a separator, the command may change the original format. The suggestion is more robust, IMO.

Reply:
@toshipp Thank you for your improvement suggestion. It looks great.

However, I think that the output of your command includes an extra null character.
Though it is actually invisible, its output looks something like this (which contains an invisible null character at the begining of the title):
```
- \0Bump chart version to 10.0.0 ([#588](https://github.com/topolvm/topolvm/pull/588))
```

I think the following command can eliminate the null character. What do you think?
```suggestion
   git log --merges --format="%s%x00%b" $(git tag | grep "^v" | sort -V -r | head -n 1)..main | sed -E 's|^Merge pull request #([0-9]*)[^\x0]*\x0(.*)|- \2 ([#\1](https://github.com/topolvm/topolvm/pull/\1))|' | tac
```

Reply:
I fixed it.